functions/icons.zsh

```diff
@@ -112,6 +112,7 @@ case $POWERLEVEL9K_MODE in
       TIME_ICON                      $'\uE12E'              # 
       JAVA_ICON                      $'\U2615'              # ☕︎
       LARAVEL_ICON                   ''
+      HASKELL_ICON                   $'\U03BB='
```

Comment:
That equal sign is there for a purpose?

Reply:
Yes, it was an attempt to get something similar to the Haskell logo character already present in Nerd Fonts, appending it to the lowercase lambda character `\u03bb` to get the symbol `λ=` in the Stack segment when other fonts are used.

If it's prone to confusion we can remove it and just use the lowercase lambda.

Reply:
Allright. Thanks for clearing that up. I thought it was a mistake. But if it's intentional, I am fine with it.